ICAFIX/ReApplyFixPipeline.sh

```diff
@@ -663,7 +686,11 @@ main()
 			$FSLDIR/bin/imrm ${StudyFolder}/${Subject}/MNINonLinear/Results/${fMRIName}/${fMRIName}${hpStr}
 		fi
 		$FSLDIR/bin/imrm ${StudyFolder}/${Subject}/MNINonLinear/Results/${fMRIName}/${fMRIName}${hpStr}.ica/filtered_func_data
-        rm ${StudyFolder}/${Subject}/MNINonLinear/Results/${fMRIName}/${fMRIName}${hpStr}.ica/Atlas_hp_preclean.dtseries.nii
+		#TSC: should these be ${hpStr} or _hp${hp}?
+		rm -f "${StudyFolder}/${Subject}/MNINonLinear/Results/${fMRIName}/${fMRIName}${hpStr}.ica/Atlas_hp_preclean.dtseries.nii" "${StudyFolder}/${Subject}/MNINonLinear/Results/${fMRIName}/${fMRIName}${hpStr}.nii.gz"
```

Comment:
L690 has a bug.  We can delete `Atlas_hp_preclean.dtseries.nii`, but `${fMRIName}${hpStr}.nii.gz` is already deleted above, and importantly, is only deleted there conditional on hp>=0.  Otherwise, if hp<0 applies, we would be deleting the _original_ volume (since `${hpStr}` will be the empty string in that condition).

Reply:
I think Matt saw and fixed this in 4eedd5ed883a2e2972bae49e4abaee5394c43c2d and 5ee0ad8fa9d29added87b5792475bf5c3d00c0df.